oo-install/site_assets/oo-install-bootstrap.sh

```diff
@@ -61,13 +61,19 @@ elif [[ $RUBYVER == ruby\ 2\.* ]]
 then
   RUBYDIR=''
 fi
-GEM_PATH="${TMPDIR}INSTALLPKGNAME/vendor/bundle/ruby/${RUBYDIR}gems/"
+
+OOINSTALL_GEM_PATH="${TMPDIR}INSTALLPKGNAME/vendor/bundle/ruby/${RUBYDIR}gems/"
+
+# Prepend the oo-install GEM_PATH to the existing GEM_PATH
+GEM_PATH=$OOINSTALL_GEM_PATH${GEM_PATH:+:$GEM_PATH}
+
 RUBYLIB="${TMPDIR}INSTALLPKGNAME/lib:${TMPDIR}oo-install/vendor/bundle"
-for i in `ls $GEM_PATH`
-do
-  RUBYLIB="${RUBYLIB}:${GEM_PATH}${i}/lib/"
+# For each directory in GEM_PATH, add any subdirectories's ./lib directories to RUBYLIB
+for path in $(find ${GEM_PATH//:/ } -maxdepth 2 -type d -name lib); do 
```

Comment:
@jrafanie I'm setting up a modified version for testing now. I'm fairly confident tests will pass. I've been testing iterations on the CFME system I was given access to by Milan.